lib/smart_answer/calculators/uk_benefits_abroad_calculator.rb

```diff
@@ -35,6 +35,12 @@ def benefits?
       benefits.present? && benefits.is_a?(Array) && valid_benefits?
     end
 
+    def dispute_criteria?
+      dispute_criteria.present? &&
```

Comment:
See [here](https://github.com/alphagov/smart-answers/pull/3180#discussion_r132136772) - we may want a helper class for this sort of thing.